compaction/compaction.cc

```diff
@@ -1279,7 +1282,9 @@ class scrub_compaction final : public regular_compaction {
         }
```

Comment:
Doesn't this inaccurately duplicate all the throws?

Maybe maybe_abort_scrub() should take as a parameter all the code that follows?

Or maybe just the three lines that follow maybe_abort_scrub()?